media/server/main/interface/IWebAudioPlayerServerInternal.h

```diff
@@ -39,6 +39,15 @@
 #include "ISharedMemoryBuffer.h"
 #include "IWebAudioPlayer.h"
 #include <MediaCommon.h>
+namespace firebolt::rialto::server
```

Comment:
fwd declarations

Reply:
Removed these forward declarations and replaced them with includes